src/main/java/eu/xenit/alfred/content/gateway/servicediscovery/AppService.java

```diff
@@ -0,0 +1,24 @@
+package eu.xenit.alfred.content.gateway.servicediscovery;
+
+public record AppService(String name, String appId, String deploymentId, String policyPackage, String namespace) {
+    // Since this is a kubernetes service, the name is guaranteed to be unique and thereby suitable as id
+    public String id() {
+        return name();
+    }
+
+    public String serviceUrl() {
+        return "http://%s.%s.svc.cluster.local:8080".formatted(name, namespace);
+    }
+    public String policyPackage() {
+        if (this.policyPackage != null) {
+            return this.policyPackage;
+        }
+        return "contentgrid.userapps.deployment" + this.deploymentId().replace("-", "");
```

Comment:
:grimacing:

Reply:
This is clearly a fallback in case it's not defined on the annotations